spectrum-network/tests/tests.rs

```diff
@@ -1,7 +1,7 @@
+mod integration_tests;
```

Comment:
Similarly, changes in this file are identical to associated changes in https://github.com/spectrum-finance/spectrum/pull/11.